mapper_test.go

```diff
@@ -287,11 +288,17 @@ mappings:
   timer_type: summary
   name: "foo"
   labels: {}
+  quantiles:
+    - quantile: 0.42
+      error: 0.04
+    - quantile: 0.7
+      error: 0.002
   `,
 			mappings: mappings{
 				"test.*.*": {
-					name:   "foo",
-					labels: map[string]string{},
+					name:      "foo",
+					labels:    map[string]string{},
+					quantiles: []metricObjective{},
```

Comment:
Why does this come back empty from the mapping? Shouldn't this contain the quantiles defined above?

Can you also add a test for the case where it's defined in the defaults?

Reply:
Done.